### PR TITLE
Create newborntownWeb adapter

### PR DESCRIPTION
**Pull request by z-sunshine**
Create newborntownWeb adapter

### CODE REVIEW

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/1605

File: modules/newborntownWebBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {BANNER, NATIVE} from '../src/mediaTypes';
+const BIDDER_CODE = 'newborntownWeb';
+
+let useSSL = document.location.protocol == 'https:';
```
Comment (sumit116) [inline modules/newborntownWebBidAdapter.js]: As a part of prebid-3.0 release (http://prebid.org/blog/pbjs-3), all adapters can call only secure endpoints, so you shall remove this line.

File: modules/newborntownWebBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+const REQUEST_URL = (useSSL ? 'https:' : 'http:') + '//us-west.solortb.com/adx/api/rtb?from=4'
```
Comment (sumit116) [inline modules/newborntownWebBidAdapter.js]: only https protocol.

File: test/spec/modules/newborntownWebBidAdapter_spec.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+          'id': '174548259807190369860081',
+          'seatbid': [
+            {
+              'bid': [
+                {
+                  'id': '1573540665390298996',
+                  'impid': '1',
+                  'price': 0.3001,
+                  'adid': '1573540665390299172',
+                  'nurl': 'http://us-west.solortb.com/winnotice?price=${AUCTION_PRICE}&ssp=4&req_unique_id=740016d1-175b-4c19-9744-58a59632dabe&unique_id=06b08e40-2489-439a-8f9e-6413f3dd0bc8&isbidder=1&up=bQyvVo7tgbBVW2dDXzTdBP95Mv35YqqEika0T_btI1h6xjqA8GSXQe51_2CCHQcfuwAEOgdwN8u3VgUHmCuqNPKiBmIPaYUOQBBKjJr05zeKtabKnGT7_JJKcurrXqQ5Sl804xJear_qf2-jOaKB4w',
```
Comment (sumit116) [inline test/spec/modules/newborntownWebBidAdapter_spec.js]: update to https.

**Comment by z-sunshine** [comment]
@sumit116 The problem has been modified. Please review again

**Comment by z-sunshine** [comment]
@sumit116 Hello, I found that [Prebid 2.42.0 Release](https://github.com/prebid/Prebid.js/releases/tag/2.42.0) has been released, but I can't find the option of newborntownWeb in Select Bidder Adapters [http://prebid.org/download.html](http://prebid.org/download.html) . Why is this?

**Comment by bretg** [comment]
docs file needs to be merged before showing up on the download page. done.

**Comment by z-sunshine** [comment]
> docs文件需要合并，然后才能显示在下载页面上。完成。

Already available for download, thanks

**Comment by z-sunshine** [comment]
> docs file needs to be merged before showing up on the download page. done.

Already available for download, thanks

**Pull request merged**